README.md

```diff
@@ -1,23 +1,71 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Sewing Pattern Maker
 
-## Available Scripts
+Our web app allows users to generate a sewing pattern from a  photo of an article of clothing.
```

Comment:
Maybe we should add the word basic, so it says "Our web app allows users to generate a basic sewing pattern..."

Reply:
I guess it would be good to slightly lower expectations, I agree.